### DISCUSSION TITLE
How many chip instances does S98 file format support?

### DISCUSSION THREAD

**Discussion by freq-mod** [Wohlstand/OPN2BankEditor#108] [Q&A] [answered]
Iirc VGM supports only 2 of them, but S98?

**Comment by jpcima** [comment, marked as answer]
S98 is able to have any number of devices of any type, and clock frequency.
It's more interesting than VGM in that aspect. (which has the limit fixed to 2)

https://github.com/Wohlstand/OPN2BankEditor/blob/3661a17bf312fc8234998ebea44e07e733167241/Specifications/s98spec3.txt#L15-L16
